Frends.SFTP.DeleteFiles/Frends.SFTP.DeleteFiles.Tests/Lib/Helpers.cs

```diff
@@ -97,46 +95,44 @@ internal static string ConvertToSHA256Hex(byte[] hostKey)
 

```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/3175f06b02a331725450b4c74d2a60e2286e92a7/Frends.SFTP.DeleteFiles/Frends.SFTP.DeleteFiles.Tests/Lib/Helpers.cs#L69
This method is not used, so it can be removed.

https://github.com/FrendsPlatform/Frends.SFTP/blob/3175f06b02a331725450b4c74d2a60e2286e92a7/Frends.SFTP.DeleteFiles/Frends.SFTP.DeleteFiles.Tests/Lib/Helpers.cs#L74
This method is not used, so it can be removed.